budget/models.py

```diff
@@ -54,7 +54,7 @@ def balance(self):
     def uses_weights(self):
         return len([i for i in self.members if i.weight != 1]) > 0
 
-    def get_transactions_to_settle_bill(self):
+    def get_transactions_to_settle_bill(self, pretty_output=False):
         """Return a list of transactions that could be made to settle the bill"""
```

Comment:
In order to have a better readability, I would create a `prettify` function (inside this method) which will do the conversion if needed.